packages/manager/modules/bm-server-components/src/ipmi/order-kvm/controller.js

```diff
@@ -11,74 +10,59 @@ export default class BmServerComponentsOrderKvmController {
 
   $onInit() {
     this.serviceName = this.server.name;
-    this.order = {};
-    this.loaders = {
-      details: false,
-      validation: false,
-    };
-    this.loadOptionDetails();
+    const [datacenter] = this.server.datacenter.split('_');
+    this.datacenter = datacenter.toLowerCase();
+    this.contractAgreement = false;
+    this.pendingOrder = false;
+    this.loading = true;
```

Comment:
**nitpick:** For better readability this can be moved inside `prepareKvmCart` function. From the initial glance, we know that we set `loading = true` but we will have to search to see where it is set to false again.

Reply:
I fixed